scripts/install-deeplearning-libraries.sh

```diff
@@ -62,18 +62,16 @@ fi
 # Build numpy from source against OpenBLAS
 # You might need to install liblapack-dev package as well
 # sudo apt-get install -y liblapack-dev
-if [ ! -d "numpy" ]; then
-    git clone -q --branch=v1.9.1 git://github.com/numpy/numpy.git
-    ln -s dl-machine/numpy-site.cfg numpy/site.cfg
-    (cd numpy && python setup.py install)
+if [ ! -d "~/.numpy-site.cfg" ]; then
```

Comment:
`-d` is for testing the presence of directories. Here `.numpy-site.cfg` is a symlink to a file. I think we need `-x` here. Could you please check that this works on a new instance?